RELEASENOTES.md

```diff
@@ -1,5 +1,12 @@
 # Release notes
 
+## 6.0.0
+
+### Features
+* Removed generation of Security Controls file and flags to customise its content. Security Controller is now hardcoded in a specific SDK version. More info can be found in the SDK documentation 
+Android - //ADD LINK
```

Comment:
https://thalesdocs.com/oip/omi-sdk/ios-sdk/ios-sdk-security-controls/index.html

Reply:
nope, this will be edited later as part of the other ticket